packages/mui-material/src/styles/createTypography.d.ts

```diff
@@ -15,7 +15,8 @@ export type Variant =
   | 'body2'
   | 'caption'
   | 'button'
-  | 'overline';
+  | 'overline'
+  | 'inherit';
```

Comment:
The `inherit` key was missing in theme's typography type - https://codesandbox.io/s/serverless-worker-7sf56k?file=/src/App.tsx.

Reply:
I don't think that we should make this change, we don't want developers to rely on it.

Reply:
I had the same thought, but shouldn't the types should match the implementation? It seems like we are shipping a bug in that case. Considering this, I believe it's best to completely remove the logic from the theme, even if it means introducing a breaking change. If a issue is reported for type error and then we are advising not to rely on this in the theme, then we could also state that we have removed it entirely. In any case, I've already removed this change.